ZIGSIMPlus/Views/CommandSelectionViewController.swift

```diff
@@ -171,40 +166,45 @@ final class CommandSelectionViewController: UIViewController {
                                         height: unlockPremiumFeatureButton.frame.size.height)
     }
     
-    private func setAlertMessage(_ message: String, _ alertType: alertModalType) {
-        var msg = message
-        if  alertType == alertModalType.premium && !isAvailablePremiumCommands() {
-            msg = premiumTextBody + "\n\nBut the following function can not be used on your device."
+    private func getAlertMessageForPurchase() -> String {
+        var message = premiumTextBody
+        if  unavailablePremiumFunctionCount > 0 {
+            message += (unavailablePremiumFunctionCount >= 2
+                ? "\n\nThe following functions don't work on this device:"
+                : "\n\nThe following function doesn't work on this device:")
```

Comment:
I think `The following` should be `Following` ...? 🤔

Reply:
Adding "The" seems to be correct.
https://ell.stackexchange.com/questions/100202/the-following-vs-following

Reply:
😳